### DISCUSSION TITLE
How to use wrapper and relativeToWrapper?

### DISCUSSION THREAD

**Discussion by afredericksansait** [scottjr632/react-rellax-wrapper#13] [Q&A]
Would it be possible to get an example because what I've been trying doesn't seem to work.

**Comment by srctl** [comment]
The example in [this](https://scottjr632.github.io/react-rellax-wrapper/?path=/story/example-horizontal--horizontal) storybook uses `relativeToWrapper`. The wrapper that the element is relative to is the parent div that is outstretched horizontally. It has the id of `wrapper` in this example. Then when using the `RellaxWrapper ` component, you should specify wrapper using css selectors (e.g. `#wrapper` if it is an id or `.wrapper` if it is a class). Then by setting `relativeToWrapper` to true, the child components of `RellaxWrapper` will move relative to the wrapper specified.
